fenix/app/src/main/java/org/mozilla/fenix/library/LibraryPageFragment.kt

```diff
@@ -32,8 +30,6 @@ abstract class LibraryPageFragment<T> : Fragment() {
                     tabsUseCases.addTab.invoke(url, private = private)
                 }
         }
-
-        (activity as HomeActivity).browsingModeManager.mode = BrowsingMode.fromBoolean(private)
```

Comment:
Just a bit dizzy from reading all the code - I am just not seeing any use case file changes in the PR. Where do we handle the browsing mode updates when it goes through the use cases?

Reply:
Through the `BrowserStoreBinding`. The tabs use cases will update the selected session id, which will propagate to a mode change in the `AppStore` through the binding.